libmaven/mzMassSlicer.cpp

```diff
@@ -29,13 +29,14 @@ void MassSlices::stopSlicing() {
 }
 /**
  * MassSlices::algorithmB This is the main function that does the peakdetection
- * This dont need a DB to check the peaks. This function finds all the peaks
```

Comment:
@Raghavdata What we need to include here is a description of algorithmB what was already there was a general description of what mass slices in general so add a specific comment here.